QUICKSTART.md

```diff
@@ -0,0 +1,362 @@
+# **KCP-Edge** User Guide
```

Comment:
This no longer agrees with the filename.

Reply:
fixed